packages/site/src/pages/index.tsx

```diff
@@ -328,15 +348,15 @@ const Index = () => {
       const txData =
         transferFunctionSelector +
         targetAccount?.slice(2).padStart(64, '0') +
-        (Number(transferAmount).toString(16)).padStart(64, '0');
+        (Number(parseUnits(transferAmount, 'ether')).toString(16)).padStart(64, '0');
```

Comment:
USDC has a different token decimals unlike ETH and Boba, so we will have to parse it differently